clientapp/src/App.css

```diff
@@ -793,4 +793,38 @@ input:checked + .slider:before {
 .share-button {
   display: inline;
   line-height: 60px;
+}
+
+.footer {
```

Comment:
There's still a bit of a vertical scrollbar. Try giving footer height of 10% so that landing page + footer = 100% of the height. And remove the padding from the footer

Reply:
I have updated footer height to 10%, and removed padding.